common/pref_names.h

```diff
@@ -74,6 +74,7 @@ extern const char kBraveShieldsSettingsVersion[];
 extern const char kBinanceAccessToken[];
 extern const char kBinanceRefreshToken[];
 extern const char kDefaultBrowserPromptEnabled[];
+extern const char kBraveHumanWebEnabled[];
```

Comment:
please add a space above and below. I know the other prefs don't, but that's a mistake. We should be grouping them

Reply:
fixed